### PR TITLE
HOTFIX (again?) | Add entry point for nav state with the menu closed

### PR DESCRIPTION
**Pull request by renanpvaz**
#### :thinking: What?
Add an actual new entry point for the nav state instead of hacking my way around it

#### :man_shrugging: Why?
Because my past "fix" forces the menu to never open :facepalm:  

### :fire: Extra
Sorry, this time it'll work

### CODE REVIEW

**Review by miguelcoba — APPROVED** [review]

**Review by kress95 — APPROVED** [review]

File: src/UI/NavigationContainer.elm [hunk 1]
```diff
@@ -49,7 +49,7 @@ Example of usage:
 
 # Model & Update
 
-@docs Msg, State, stateInit, stateUpdate
+@docs Msg, State, stateInit, stateInitWithClosedMenu, stateUpdate
```
Comment (PedroHLC) [inline src/UI/NavigationContainer.elm:52]: :thinking: why not a proper `stateWithMenuOpen : Boolean -> State -> State`?

Comment (renanpvaz) [reply src/UI/NavigationContainer.elm:52]: because so far there's no need for a state that is forced open

Comment (PedroHLC) [reply src/UI/NavigationContainer.elm:52]: then, what about `stateInit |> stateWithMenuClosed` ?

Comment (renanpvaz) [reply src/UI/NavigationContainer.elm]: done

**Review by PedroHLC — APPROVED** [review]

**Pull request merged**